### PR TITLE
Amends to LowerESIMD.cpp intrinsics table

### PR DESCRIPTION
**Pull request by NikitaRudenkoIntel**

### CODE REVIEW

**Review by DenisBakhvalov — CHANGES_REQUESTED** [review]
The change LGTM, but please add tests (`llvm/test/SYCLLowerIR/esimd_lower_intrins.ll`).

File: llvm/lib/SYCLLowerIR/LowerESIMD.cpp
Comment (kbobrovs) [inline llvm/lib/SYCLLowerIR/LowerESIMD.cpp:1063]: Is it possible to add a test? Otherwise LGTM, as long as Denis's comments are addressed

**Comment by kbobrovs** [comment]
> @bader bader requested review from DenisBakhvalov and kbobrovs 6 minutes ago

There are still no tests, I can't approve yet.

**Comment by NikitaRudenkoIntel** [comment]
I will add them as soon as I fix existing failures.

File: llvm/lib/SYCLLowerIR/LowerESIMD.cpp
Comment (NikitaRudenkoIntel) [reply llvm/lib/SYCLLowerIR/LowerESIMD.cpp:1067]: Added test

**Review by kbobrovs — APPROVED** [review]

**Comment by kbobrovs** [comment]
already reviewed in internal codebase

**Comment by kbobrovs** [comment]
@DenisBakhvalov - is a change you requested implemented yet?

**Comment by kbobrovs** [comment]
@DenisBakhvalov, ping

**Review by DenisBakhvalov — APPROVED** [review]

**Pull request merged**
